webpack.config.cjs

```diff
@@ -232,11 +232,35 @@ const templateContent = ({ htmlWebpackPlugin }) => {
     })
     .filter(omitPrimaryChunk);
 
-  return `<?php return array(
-    'css' => ${JSON.stringify(css)},
-    'js' => ${JSON.stringify(js)});`;
+  // We're only interested in chunks from dynamic imports;
+  // ones that are not already in `js` and not primaries.
+  const chunks = chunkNames.filter(
+    (chunk) =>
+      !js.includes(chunk) &&
+      !['wp-dashboard', 'wp-story-editor'].includes(chunk)
```

Comment:
Nit: would be nice to dedupe the `wp-dashboard` and `wp-story-editor` strings now that they're used outside of the single object literal below.

```js
const EDITOR_CHUNK = 'wp-story-editor';
const DASHBOARD_CHUNK = 'wp-dashboard';
```